core/encoding.php

```diff
@@ -210,12 +210,14 @@ function iconv_array($source_encoding, $target_encoding, $data)
         // recursive call for array conversion
         foreach ($data as $key => $val)
         {
+            if (is_null($val)) {$val = '';} // Null not allowed
             $data[$key] = iconv_array($source_encoding, $target_encoding, $val);
         }
     }
     else
     {
         // finally convert string value
+        if (is_null($data)) {$data = '';}  // Null not allowed
```

Comment:
removing this line and using a cast on line 221 `(string) $data` will keep the functionality of line 222 while fixing the null-not-allowed error...

Reply:
i am  not familiar with using a cast

Reply:
code amended to cast string